.jenkins/common.groovy

```diff
@@ -7,7 +7,7 @@ def runCompileCommand(platform, project, jobName)
         
     def command 
 
-    def getRocPRIM = auxiliary.getLibrary('rocPRIM', platform.jenkinsLabel,'develop')
+    def getRocPRIM = auxiliary.getLibrary('rocPRIM', platform.jenkinsLabel,'${env.BRANCH_NAME}')
```

Comment:
I need to check this. @eidenyoshida and @amdkila we should use this option in the future so hipCUB master always tests with hipCUB master, not develop.

Reply:
Should be relevant to many libraries.